src/silx/gui/widgets/FrameBrowser.py

```diff
@@ -294,13 +329,17 @@ def __init__(self, parent=None):
         self._playButton.setToolTip("Display movie with frames")
         self._playButton.setIcon(icons.getQIcon("camera"))
         self._playButton.setIconSize(iconSize)
+        self._playButton.setCheckable(True)
         self.mainLayout.addWidget(self._playButton)
         
-        self._playButton.clicked.connect(self._playStopSequence)
+        self._playButton.toggled.connect(self._playButtonToggled)
```

Comment:
Use `toggled` rather than `clicked` so it is also called when set programmatically.